lib/danger/request_sources/github.rb

```diff
@@ -174,12 +192,12 @@ def parse_comment(comment)
         violations[kind] = violations_from_table(table)
       end
 
-      violations
+      violations.reject { |k, v| v.empty? }
     end
 
     def violations_from_table(table)
-      regex = %r{<td data-sticky="true">(<strike>)?(.*?)(</strike>)?</td>}im
-      table.scan(regex).flatten.compact.map(&:strip)
+      regex = %r{<td data-sticky="true">(<del>)?(.*?)(</del>)?\s*</td>}im
+      table.scan(regex).map { |a| a[1] }.map(&:strip)
```

Comment:
If we're only using the second match, we could simplify this (without 2 maps) by using non-capturing groups like this:

`<td data-sticky="true">(?:<del>)?(.*?)(?:</del>)?\s*</td>`